Program.cs

```diff
@@ -0,0 +1,84 @@
+using System.Text;
+using Amazon;
+using Amazon.DynamoDBv2;
+using Amazon.Runtime;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.IdentityModel.Tokens;
+using Movies.Api;
+using Movies.Api.Repositories;
+using Movies.Api.Utils;
+using Newtonsoft.Json;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+builder.Services.AddControllers();
+builder.Services.AddAWSService<IAmazonDynamoDB>();
+
+var credentialsFilePath = "aws_credentials.json";
+var json = File.ReadAllText(credentialsFilePath);
+var credentials = JsonConvert.DeserializeObject<AwsCredentials>(json);
+
+builder.Services.AddSingleton<IAmazonDynamoDB>(new AmazonDynamoDBClient(
+    new BasicAWSCredentials(credentials.AwsAccessKeyId,credentials.AwsSecretAccessKey), // Add your AWS credentials
+    new AmazonDynamoDBConfig
+    {
+        ServiceURL = "https://dynamodb." + credentials.Region + ".amazonaws.com" // Optional for explicitly pointing to the service URL
+    }
+));
+
+builder.Services.AddTransient<IUserRepository, UserRepository>();
+builder.Services.AddTransient<IMovieRepository, MovieRepository>();
+builder.Services.AddTransient<IReviewRepository, ReviewRepository>();
```

Comment:
I created a class to handle dependency injections. It was done by ChatGPT mostly, because I'm a bit rusty with OOP (we mostly use functional programming in the frontend)